README.md

```diff
@@ -20,9 +20,54 @@ Via Composer
 $ composer require php-http/httplug-bundle
 ```
 
+Enable the bundle in your kernel:
+ 
+``` php
+<?php
+// app/AppKernel.php
+
+public function registerBundles()
+{
+    $bundles = array(
+        // ...
+        new Http\HttplugBundle\HttplugBundle(),
+    );
+}
+```
 
 ## Usage
 
+The usage documentation is split into two parts. First we explain how to configure the bundle in an application. The second part is for developing reusable Symfony bundles that depend on an HTTP client defined by the Httplug interface.
+
+### Use in applications
+
+This bundle provides 3 services: 
+
+* `httplug.client` a service that provides the `Http\Client\HttpClient`
```

Comment:
depending on the decision on https://github.com/php-http/httplug/issues/68#issuecomment-149112039 this might need to be changed